frame/assets/src/lib.rs

```diff
@@ -186,6 +185,17 @@ pub mod pallet {
 	#[pallet::storage_version(STORAGE_VERSION)]
 	pub struct Pallet<T, I = ()>(_);
 
+	#[cfg(feature = "runtime-benchmarks")]
+	pub trait BenchmarkHelper<AssetId> {
+		fn create_asset_id(id: u32) -> AssetId;
+	}
+	#[cfg(feature = "runtime-benchmarks")]
+	impl<AssetId: From<u32>> BenchmarkHelper<AssetId> for () {
+		fn create_asset_id(id: u32) -> AssetId {
```

Comment:
Shall we follow the uniques example and name it `fn asset(id: u32) -> AssetId {`?
Especially given we don't actually create an asset here

Reply:
We can, but I thought `create_asset_id` expresses that it's creating an ID that conforms to `T::AssetId`, not an asset or asset class.